packages/firestore/src/local/simple_db.ts

```diff
@@ -494,7 +510,7 @@ export class SimpleDbStore<KeyType extends IDBValidKey, ValueType> {
  * Wraps an IDBRequest in a PersistencePromise, using the onsuccess / onerror
  * handlers to resolve / reject the PersistencePromise as appropriate.
  */
-function wrapRequest<R>(request: IDBRequest): PersistencePromise<R> {
+export function wrapRequest<R>(request: IDBRequest): PersistencePromise<R> {
```

Comment:
As mentioned elsewhere I'd like to keep this abstracted away from consumers of SimpleDb and not export it.

Reply:
I reverted the export.